### PR TITLE
fix: Remove min-height on small screens to prevent card overflow off …

### PR DESCRIPTION
**Pull request by mcstover**
…the screen.

### CODE REVIEW

File: src/components/Contentful/StoryCard.vue [hunk 1]
```diff
@@ -101,11 +101,11 @@ export default {
 	.story-card__imageCard {
 		aspect-ratio: 4 / 5;
 		border-radius: 2.5rem;
-		min-height: 550px;
 		overflow: hidden;
 		padding: 2.5rem 1.25rem;
 		@include breakpoint(medium) {
 			padding: 2.5rem;
+			min-height: 550px;
```
Comment (mcstover) [inline src/components/Contentful/StoryCard.vue:108]: This is too tall for mobile devices causing the cards to overlap and overflow the viewport.

image

With Fix:
image

**Comment by mcstover** [comment]
This is for a hotfix tonight.

**Comment by mcstover** [comment]
merging to Verify in dev

**Comment by eddieferrer** [comment]
Looks good!

**Review by eddieferrer — APPROVED** [review]

**Pull request merged**